middlewares/auth_required.js

```diff
@@ -1,3 +1,10 @@
+import { useAuth } from '../context/auth'
 /***
  * @todo Redirect the user to login page if token is not present.
  */
+const checkLogin = () => {
+    // console.log(document.cookie); 
+    if(document.cookie=='')window.location.href = './login'
```

Comment:
So I have to use router instead of href and rather than checking document.cookie again and again I should use State ??

Reply:
So I have to use router instead of window.location.href and rather than checking document.cookie again and again I should use State ??
Is that all or I am still missing something?

Reply:
Yes

Reply:
I made a few changes to resolve those mistakes. Please review them.